pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -480,6 +480,18 @@ func maybeRevertToCutoverTimestamp(
 
 	updateRunningStatus(ctx, j, fmt.Sprintf("starting to cut over to the given timestamp %s", cutoverTime))
 
+	err = j.NoTxn().Update(ctx, func(txn isql.Txn, md jobs.JobMetadata, ju *jobs.JobUpdater) error {
```

Comment:
Should we put the above validations in the same transaction as the state update?

Reply:
done, makes sense.